datatypes/src/main/java/org/hyperledger/besu/datatypes/Address.java

```diff
@@ -75,6 +80,16 @@ public class Address extends DelegatingBytes {
   /** The constant ZERO. */
   public static final Address ZERO = Address.fromHexString("0x0");
 
+  static LoadingCache<Address, Hash> hashCache =
```

Comment:
we don't   want to add a limit for this cache ?

Reply:
What limit would you suggest?

Reply:
I would say a maximum size. Maybe we should see how much on average we load hash by block and got a number. But with a cache without max seems dangerous to me . Or else there is a default limit?